lib/schedule/schedule.dart

```diff
@@ -17,8 +17,25 @@ class ScheduleView extends StatelessWidget {
         ]
     );
 
+    final emptyState = new Container(child:
+      new Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            new Image.asset("assets/nothing_here.png"),
+            new Text('Nothing here. Come back soon!',
+              textAlign: TextAlign.center,
+              style: theme.textTheme.display1.copyWith(color: Colors.black, fontSize: 16.0),),
+          ],
+      )
+    );
+
+    final body = new PageView(children: <Widget>[
+      emptyState,
+    ],);
```

Comment:
is the comma intended? not sure if it compiles cause there's no CI yet 👅

Reply:
The trailing commas thing baffles me too, not entirely sure if it's needed, will check!

Reply:
It's needed for `dartfmt` and the IDE to know how to format lists correctly and is a common practice in Flutter code